### ISSUE TITLE
"checkbox" column with "switchable: true" not listed in "showColumns" column picker

### ISSUE THREAD

**Issue by tomaszn** [wenzhixin/bootstrap-table#6825]
### Description

Hello,

Is there a reason that the column with checkboxes or radio buttons [is not displayed in the column picker](https://github.com/wenzhixin/bootstrap-table/blob/7ba6c41e0122774caa219e7be46971c3479dcb74/src/bootstrap-table.js#LL713C12-L713C12), so can't be toggled by the user?

### Example(s)

<table
  id="table"
  data-toggle="table"
  data-height="460"
  data-show-columns="true"
  data-click-to-select="true"
  data-pagination="true"
  data-maintain-meta-data="true"
  data-url="json/data1.json">
  <thead>
    <tr>
      <th data-field="state" data-checkbox="true"></th>
      <th data-field="id">ID</th>
      <th data-field="name">Item Name</th>
    </tr>
  </thead>
</table>

**Comment by tomaszn** [comment]
@c-viswanath Thank you for your response. My example could confuse you, sorry. Some table attributes didn't matter in my example, and `data-click-to-select` is one of them.

My issue is about the dropdown with a column picker on the right, above the table. I can't set it up to show also the first column with checkboxes.

I made a better example, without unnecessary attributes:
https://live.bootstrap-table.com/code/tomaszn/15648

**Comment by c-viswanath** [comment]
Ohhh ok, gotcha...you need the first column to also show up in the dropdown right? I'll try to look into it and let you know if I find any solution for the same.

**Comment by tomaszn** [comment]
@c-viswanath Yes, exactly. The purpose is to enable a "Bulk editing mode" simply by enabling the column with checkboxes.
